### DISCUSSION TITLE
Row grouping for list operation?

### DISCUSSION THREAD

**Discussion by cod3rshotout** [Laravel-Backpack/community-forum#1206] [Q&A (Help)] [answered]
Hello everyone, I have a model with the following structure:
Process -> Phases -> Activities.
I need to display all the phases of the process in a grid, each phase corresponds to multiple activities. I was wondering if there is a function like this: https://datatables.net/examples/advanced_init/row_grouping.html
if not, how can I do this? It's urgent.

**Comment by karandatwani92** [comment, marked as answer]
Hey @cod3rshotout 

The backpack doesn't have something built-in for that 😐. I think it will be easier to have a normal table and have a [filter](https://backpackforlaravel.com/docs/6.x/crud-filters#filter-types-1) to show only the entries the user wants.

**Reply by cod3rshotout** [reply]
Hi @karandatwani92 thanks for the answer, could you please take a look at this issue?
https://github.com/Laravel-Backpack/CRUD/issues/5699
